### DISCUSSION TITLE
Methods marked with ClassInitializeAttribute don't be called in Runtime Test

### DISCUSSION THREAD

**Discussion by workgroupengineering** [unoplatform/uno#11142] [Q&A]
Methot like this:

https://github.com/unoplatform/uno/blob/777ea349a80045867df62c90be9f2e33f0e259e0/src/Uno.UI.RuntimeTests/IntegrationTests/dxaml/controls/commandbar/CommandBarIntegrationTests.cs#L41-L45

never be called in Runtime Test. Is this behavior wanted?

**Comment by Youssef1313** [comment]
Looks like this method doesn't do anything.

https://github.com/unoplatform/uno/blob/73417f5c83056dbc1c33a3e5d73114aa1427162a/src/Uno.UI.RuntimeTests/IntegrationTests/common/CommonTestSetupHelper.cs#L7-L14

https://github.com/unoplatform/uno/blob/73417f5c83056dbc1c33a3e5d73114aa1427162a/src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.cs#L62

But it's unlikely we are going to change code in this file.
